docs/quickstart.rst

```diff
@@ -61,11 +61,10 @@ ignored for now.)
 Preview your site
 -----------------
 
-Open a new terminal session and run the following commands to switch to your
-``output`` directory and launch Pelican's web server::
+Open a new terminal session and run the following commands to launch Pelican's
+web server::
 
-    cd ~/projects/yoursite/output
-    python -m pelican.server
+    pelican -l content
```

Comment:
I just tried to replicate the commend 10 lines above that line where it also tells the user to run `pelican content`. So I think your first two remarks then equally apply to that existing line, no?

Your third remark is justified. I'll fix this.